entrypoint.sh

```diff
@@ -8,6 +8,12 @@ if [ -z "${JEKYLL_PAT}" ]; then
   exit 1
 fi 
 
+#Adding input variable for site and GemFile under sub directory
+if [ -n "${INPUT_GEM_SRC}" ]; then
+  cd ${INPUT_GEM_SRC}
```

Comment:
Changing directory here may yield problems with caching, see [comments](https://github.com/helaili/jekyll-action/pull/32#discussion_r426255446) in #32. Using a `--gemfile` option in the `bundle install` below and not changing directory here, or configuring the install path to `~/vendor/bundle` may resolve this

Reply:
Okay, my bad, now I got your point. Of course you are right.

Reply:
Instead of `~` we can use an [environment variable](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables) `${GITHUB_WORKSPACE}/vendor/bundle` for the bundle install, then this should fix the caching while still being able to change directory

Reply:
I guess setting both properties is a viable configuration, though not as obvious to use, isn't it?